modules/authentication/snip-common-authentication-options.adoc

```diff
@@ -0,0 +1,28 @@
+`includeTransitiveGroupOwnership`::
```

Comment:
I think it might be better to focus on the user-facing result/actual use case of this config, which is the change in the behaviour of the `$ownerRefs` alias in RBAC. 

 Imo this would fit better under the Conditional policy aliases section [here](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.4/html/authorization/con-rbac-conditional-policies-rhdh_title-authorization#con-rbac-conditional-policies-rhdh_title-authorization).

Reply:
I think I see what you mean. But

* `includeTransitiveGroupOwnership` with Authentication
  * (+) User goes through all authentication options in the same place.
  * (-) Configuration item is far way from the doc explaining the consequences of the option.


* `includeTransitiveGroupOwnership` with `$ownerRefs` alias in RBAC
	* (+) `$ownerRefs` alias behavior close to how to modify it.
  * (-) the user context is using the app, no reconfiguration of the app-config file involved.